### DISCUSSION TITLE
Welcome to SmartIR Discussions!

### DISCUSSION THREAD

**Discussion by litinoveweedle** [litinoveweedle/SmartIR#160] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by carleoneman** [comment]
After 2024.12 HA all of our custom component smartIR STOPPED!
What can be done to follow python update?!?

:::Examples:::
Platform error 'media_player' from integration 'smartir' - Requirements for smartir not found: ['aiofiles==0.6.0'].
Platform error 'climate' from integration 'smartir' - Requirements for smartir not found: ['aiofiles==0.6.0'].
Platform error 'fan' from integration 'smartir' - Requirements for smartir not found: ['aiofiles==0.6.0'].

In mind that many of us don't want to convert smartir commands to direct broadlink integrations, etc.

**Reply by litinoveweedle** [reply]
1. If you have an issue, open the issue.
2. When you open issue follow the issue template - otherwise I will not take care.
3. You are probably even not using this project but the original one.
4. You are seems to be confused by some code conversion, which seems to not be related in any way.
5. Your comment seems to be a bit rude, if you seek help watch your attitude.
